Doc/whatsnew/3.8.rst

```diff
@@ -168,6 +168,16 @@ asyncio
 On Windows, the default event loop is now :class:`~asyncio.ProactorEventLoop`.
 
 
+ctypes
+------
+
+On Windows, :class:`~ctypes.CDLL` and subclasses now accept a *winmode* parameter
+to specify flags for the underlying ``LoadLibraryEx`` call. The default flags are
+set to only load DLL dependencies from trusted locations, including the path
+where the DLL is stored (if a full or partial path is used to load the initial
+DLL) and paths added by :func:`~os.add_dll_directory`.
+
+
```

Comment:
The last sentence could be merged with `ctypes.rst`

Reply:
It could, but I'm trying to avoid this being part of the permanent contract (in case Windows comes up with a new way in the future to deal with this). For porting to 3.8 it's fine to assume it, but the ctypes docs are deliberately more vague about how the OS mechanics work.